lib/active_record/annotate/file.rb

```diff
@@ -19,39 +20,46 @@ def annotate_with(annotation, configurator)
           # separate magic comments from the annotation with an empty line
           magic_comments << nil
         end
-        
+
         while @lines.first.start_with?('#') || @lines.first.blank?
           # throw out comments and empty lines
           # in the beginning of the file (old annotation)
           @lines.shift
         end
-        
-        if configurator.yard?
-          backticks = '# ```'
-          annotation.unshift(backticks).push(backticks)
+
+        if annotation.first.start_with?("# create_table")
```

Comment:
@Morozzzko will this work for views, or we need something like `annotation.first.start_with?("# create_table") || annotation.first.start_with?("# create_view")`? Can you tell us how the views look like in your app's annotations?

Reply:
I'll check and tell you by Tuesday evening

Reply:
Sorry I'm late. It says `create_table` even if it's a view

Probably because it's a materialized view, not sure